library/psa_crypto_storage.c

```diff
@@ -399,6 +405,60 @@ psa_status_t psa_load_persistent_key( psa_key_file_id_t key,
     return( status );
 }
 
+
+
+/****************************************************************/
+/* Transactions */
+/****************************************************************/
+
+#if defined(PSA_CRYPTO_STORAGE_HAS_TRANSACTIONS)
+
+psa_crypto_transaction_t psa_crypto_transaction;
+
+psa_status_t psa_crypto_save_transaction( void )
```

Comment:
Is it possible to provide an alternative implementation of transaction storage, for example for targets that will have a custom persistent memory mechanism (cloud/EEPROM/etc)?

Reply:
I guess it is, but I haven't seen any request for it. If you want an alternative mechanism, you'd only need to replace the load/save functions, so the architecture of the code supports it. Making an alternative mechanism pluggable without editing the source code is way out of scope here, especially with no concrete use case.